### PR TITLE
SIG Apps Annual report 2022

### LINKED ISSUE #7098: 2022 Annual Report: SIG Apps
Chairs: @janetkuo @kow3ns @soltysh 
Liaison: @mrbobbytables

Actions for the chair/organizer of the community group:
- [ ] Consult your community group to complete draft of report
- [ ] If needed, consult with Steering Committee liaison on any private issues or concerns that arrise while completing report
- [ ] Submit PR to git.k8s.io/community repo with copy of report, assigning your Steering Committee liaison for review
- [ ] Steering Committee reviews report, and may make comments/ask questions
- [ ] Once all comments are addressed, Steering Committee will approve report to merge
- [ ] If needed, any follow up items may be brought to the "Chairs, Tech Leads, and Organizers" meeting in April

Once all the above items are complete, this issue may be `/close`'d

A template has already been generated for your group, and is available [here](https://git.k8s.io/community/sig-apps/annual-report-2022.md).

Key dates:
- Initial PR to communtiy repo should be opened by March 24th, 2023
- PR should be reviewed and merged by April 7th, 2023

More detailed information on the annual reports process is available [here](https://git.k8s.io/community/committee-steering/governance/annual-reports.md).

Your group's report template is available here: https://git.k8s.io/community/sig-apps/annual-report-2022.md

If you have any questions or concerns about this process, you may reach the Steering Committee via the following methods:
- Slack/E-mail your liaison directly
- The public #steering-committee channel in Slack
- The public mailing list steering@kubernetes.io
- The private mailing list steering-private@kubernetes.io for any private or sensitive issues.

/sig apps
/committee steering
/area annual-reports

### PR DESCRIPTION
**Pull request by soltysh**
**Which issue(s) this PR fixes**:
Fixes #7098

/cc @janetkuo @kow3ns 
/assign @mrbobbytables

### CODE REVIEW

**Review by janetkuo — APPROVED** [review]
/lgtm

File: sig-apps/annual-report-2022.md [hunk 1]
```diff
@@ -4,81 +4,73 @@
...
-
 ## Project health
 
 1. What areas and/or subprojects does your group need the most help with?
    Any areas with 2 or fewer OWNERs? (link to more details)
 
-   -
-   -
-   -
+   - The mentoring cohort we've run in 2022 attracted a few new members, but we still need more.
```
Comment (mrbobbytables) [inline sig-apps/annual-report-2022.md:36]: Is there an issue/repo that would be good to direct folk to? these get consolidated and shared out with the GB / end users / member companies, having a place to direct potential folks would be good to include

File: sig-apps/annual-report-2022.md [hunk 2]
```diff
@@ -4,81 +4,73 @@
...
 
-   -
+   - [13 companies](https://k8s.devstats.cncf.io/d/8/company-statistics-by-repository-group?orgId=1&var-period=y&var-metric=contributions&var-repogroup_name=SIG%20Apps&var-repo_name=kubernetes%2Fkubernetes&var-companies=All&from=1609455600000&to=1639350000000) contributed code in 2022.
 
 6. Are there ways end users/companies can contribute that they currently are not?
    If one of those ways is more full time support, what would they work on and why?
 
-   -
-   -
+   - Increase the test coverage for all controllers, is probably the best way to help and start contributing to sig-apps owned controllers.
```
Comment (mrbobbytables) [inline sig-apps/annual-report-2022.md:59]: Can you link out to a few of those places? just thinking follow up again

**Review by mrbobbytables — COMMENTED** [review]
Looking good so far :) just 2 small suggestions

File: sig-apps/annual-report-2022.md (hunk 1 above)
Comment (soltysh) [reply sig-apps/annual-report-2022.md:36]: I've added link to the previous one, but I need to talk to several folks during KubeCon next week how to solve the limitations I've run into with this.

File: sig-apps/annual-report-2022.md (hunk 2 above)
Comment (soltysh) [reply sig-apps/annual-report-2022.md:59]: I don't have an issue per se, but I've added a link to a proposal that Wojtek posted some time ago. This is usually something I'm personally checking with PRs touching controllers, to ensure we don't regress but rather every PR should increase the coverage for the package its touching.

**Review by mrbobbytables — COMMENTED** [review]
/approve
Thanks!

**Pull request merged**